docs/modules/deploy/pages/installing-using-docker.adoc

```diff
@@ -1,192 +1,135 @@
+
+- Host network mode (Linux only)
+- Port mapping
 
-These commands pull Hazelcast Docker image and start a new Hazelcast member.
+WARNING: These methods open Hazelcast members to the public. Anybody who can reach the Docker host over the network can connect to Hazelcast ports of the running members.
 
-On Linux, you can omit the `-p 5701:5701` part, but on other platforms
-you may need it if the container's network isn't directly exposed to the
-host machine.
+=== Host Network Mode
 
-You should see a log on your terminal similar to this:
+To allow members and clients on different hosts to discover each other with multicast, you can use the `host` network mode. This mode allows members to share the host’s network stack from their Docker containers.
 
-[source,plain]
+[source,bash,subs="attributes+"]
 ----
-Members {size:1, ver:1} [
-	Member [172.17.0.2]:5701 - c0640efd-02ae-4595-aace-d944e1e1ad97 this
-]
+docker run --rm --network host --name member1 hazelcast/hazelcast:{full-version}
+docker run --rm --network host --name member2 hazelcast/hazelcast:{full-version}
 ----
 
-The command line prompt doesn't return, which allows you to easily stop or restart
-Hazelcast. Note Hazelcast's IP address, you'll need it later.
+In this mode, each member is available on the internal (private) IP address of the Docker host.
 
-Now, you have a single Hazelcast member.
-To ensure your codes will work properly, you should test
-it locally in a cluster with multiple members as well; testing on a single member hides some
-common serialization issues your code may have. Let's start another
-Hazelcast container to have a second member so that a 2-member cluster will be formed.
-For this, run `docker run` from another terminal window, using a
-different port mapping than the first member you created:
+NOTE: Members will only discover and connect to each other automatically if your local network supports multicast. If your network does not support multicast, you can try <<port-mapping, port mapping>>.
 
-[tabs] 
-==== 
-hazelcast:: 
-+ 
--- 
+=== Port Mapping
 
-[source,shell]
-----
-docker run -p 5702:5701 hazelcast/hazelcast:$HAZELCAST_VERSION
-----
---
+To allow members and clients on different hosts to discover each other over TCP/IP, you can map ports from the Docker host (the device that's running Docker) and tell Hazelcast to listen to your public IP address.
 
-hazelcast-enterprise::
-+
---
-[source,shell]
-----
-docker run -e 5702:5701 HZ_LICENSE_KEY=<Your Enterprise License Key> hazelcast/hazelcast-enterprise:$HAZELCAST_VERSION
-----
---
+TIP: You'll need the public IP address of your Docker host. You can find it on this web page: link:https://www.whatismyip.com/[https://www.whatismyip.com].
```

Comment:
This is only valid if the machine is directly connected to the internet. It'll probably not be the case in companies or home networks, where they usually have LANs with NAT.